lib/Map/LeafletVisualizer.js

```diff
@@ -243,15 +243,22 @@ LeafletGeomVisualizer.prototype.getLatLngBounds = function() {
 
 var cartographicScratch = new Cartographic();
 
-function positionToLatLng(position) {
+function positionToLatLng(position, prevLayer) {
   var cartographic = Ellipsoid.WGS84.cartesianToCartographic(
     position,
     cartographicScratch
   );
-  return L.latLng(
-    CesiumMath.toDegrees(cartographic.latitude),
-    CesiumMath.toDegrees(cartographic.longitude)
-  );
+
+  let lon = CesiumMath.toDegrees(cartographic.longitude);
+  if (prevLayer) {
+    if (prevLayer._latlng.lng - lon > 180) {
```

Comment:
Can we avoid digging into private fields (`_latLng`)? There's no guarantee the next Leaflet version won't remove or change a private field like that.

Reply:
I'm going to replace it with `getLatLng()` (and also check to make sure that exists). Let me know if you see any problems with that.

Reply:
Eh on second thought we seem to be using `_latLng` everywhere already. So nevermind, moving on. :)